wagtail/wagtailembeds/rich_text.py

```diff
@@ -26,6 +26,6 @@ def expand_db_attributes(attrs, for_editor):
         representation.
         """
         if for_editor:
-            return format.embed_to_editor_html(attrs['url'])
+            return format.embed_to_editor_html(attrs['url'], ignore_errors=True)
```

Comment:
Wouldn't it be a bit nicer to let ``embed_to_editor_html`` raise its error and catch it here?